eth/utils.go

```diff
@@ -892,7 +884,7 @@ func parseSubmitInput(input string) (string, string) {
 		re := regexp.MustCompile(`.{64}`) // Every 64 chars
 		parts := re.FindAllString(in, -1) // Split the string into 64 chars blocks.
 
-		if len(parts) == 4 {
+		if len(parts) >= 4 {
```

Comment:
magic number 4?
please, move it to const with readable name to be aware what's going on